client/asset/eth/eth.go

```diff
@@ -752,6 +752,24 @@ func (eth *ExchangeWallet) Redeem(form *asset.RedeemForm) ([]dex.Bytes, asset.Co
 	return inputs, outputCoin, fundsRequired, nil
 }
 
+// RecoverPubkey recovers the uncompressed public key from the signature and the
+// message hash that wsa signed. The signature should be a compact signature
+// generated by a geth wallet, with the format [R || S || V], with the recover
+// bit V at the end. See go-ethereum/crypto.Sign.
+func RecoverPubkey(msgHash, sig []byte) ([]byte, error) {
```

Comment:
It just hit me that this is exported. If will use it somewhere else, maybe goes in dex/networks/eth?

Reply:
Good point.  I suspected we'd want to use it server-side, but perhaps not.  Will just unexport for now.

For any VSCode users, this can go in settings.json to ensure the correct code is tested:

```
    "go.testEnvVars": {
        "CGO_ENABLED": 0,
    },
```